README.md

```diff
@@ -74,12 +74,12 @@ your distribution:
     ```json
     {
       "scripts": {
-        "codegen": "graph codegen --output-dir types/",
+        "codegen": "graph codegen --output-dir types/" ,
         "build": "graph build",
         "build-ipfs": "graph build --ipfs /ip4/127.0.0.1/tcp/5001",
         "deploy":
-          "graph deploy --ipfs /ip4/127.0.0.1/tcp/5001 --node http://127.0.0.1:8020 --subgraph-name <SUBGRAPH_NAME>"
-      }
+          "graph deploy --ipfs /ip4/127.0.0.1/tcp/5001 --node http://127.0.0.1:8020 --subgraph-name <SUBGRAPH_NAME>",
```

Comment:
Just noticing the comma at the end here: that's broken JSON. The last element in an array or object/map must not have a trailing comma.